packages/phrases/src/locales/tr-tr/translation/admin-console/applications.ts

```diff
@@ -11,7 +11,7 @@ const applications = {
   select_application_type: 'Uygulama tipi seçiniz',
   no_application_type_selected: 'Henüz bir uygulama tipi seçmediniz',
   application_created:
-    '{{name}} Uygulaması başarıyla oluşturuldu! \nŞimdi uygulama ayarlarını tamamlayın.',
+    '{{name}} Uygulaması başarıyla oluşturuldu. \nŞimdi uygulama ayarlarını tamamlayın.',
```

Comment:
Ditto for this phrase, would you like to also check the remaining languages? Thanks